.github/workflows/documentation-links.yml

```diff
@@ -0,0 +1,24 @@
+name: Read the Docs PR preview
```

Comment:
Somewhat unrelated to this specific PR, but I think it would be useful to add a comment (or a docstring-like description if there is a field for it) to document what the workflow is for.  If it's not possible to do it here, we could document the workflows either in a README in the dir or somewhere in the devguide.

Reply:
Good suggestion @ezio-melotti